app/components/common/Footer.vue

```diff
@@ -263,52 +270,22 @@ export default Vue.extend({
 footer#site-footer.small(:class="/^\\/(league|play\\/ladder)/.test(document.location.pathname) ? 'dark-mode' : ''" @click="footerEvent")
   .container(v-if="!hideFooter")
     .row
-      .col-lg-12
-        .row
-          .col-lg-3(v-for="col in footerUrls" v-if="col.condition" :class="!col.lists.length ? 'shrunken-empty-column' : ''")
+      .col-lg-12.footer-links
+        .row.footer-links__row
+          .col.footer-links__col(v-for="col in footerUrls" v-if="col.condition" :class="!col.lists.length ? 'shrunken-empty-column' : ''")
             h3 {{ $t(col.title) }}
             ul.list-unstyled
               li(v-for="l in col.lists" v-if="!l.hide")
-                a(:href="l.url" v-bind="l.attrs") {{ $t(l.title) }}
+                a(v-if="!checkLocation(l.url)" :href="l.url" v-bind="l.attrs") {{ $t(l.title) }}
+                  span.spr(v-if="l.extra") {{ l.extra }}
+                span.active(v-if="checkLocation(l.url)") {{ $t(l.title) }}
                   span.spr(v-if="l.extra") {{ l.extra }}
+
               li(v-if="col.title === 'nav.general'")
                 mklog-ledger(v-pre organization='org-2F8P67Q21Vm51O97wEnzbtwrg9W' kind='popper')
                   a(href="#changelog")
                     span Changelog
                     mklog-since-last-viewed(v-pre organization='org-2F8P67Q21Vm51O97wEnzbtwrg9W', color="candy")
-          .col-lg-3
-            template(v-if="!me.showingStaticPagesWhileLoading() && me.useSocialSignOn()")
-              h3 {{ $t("nav.follow_us") }}
-              div.social-buttons
-                a(href="https://www.youtube.com/channel/UCEl7Rs_jtl3hcbnp0xZclQA" target="_blank" data-event-action="Click: Footer Youtube")
-                  img(src="/images/pages/base/youtube_symbol_button.png" width="40" alt="YouTube")
-                a(href="https://twitter.com/codecombat" target="_blank" data-event-action="Click: Footer Twitter")
-                  img(src="/images/pages/base/twitter_logo_btn.png" width="40" alt="Twitter")
-                a(href="https://www.facebook.com/codecombat" target="_blank" data-event-action="Click: Footer Facebook")
-                  img(src="/images/pages/base/facebook_logo_btn.png" width="40" alt="Facebook")
-                a(href="https://www.instagram.com/codecombat/" target="_blank" data-event-action="Click: Footer Instagram")
-                  img(src="/images/pages/base/instagram-logo.png" width="40" alt="Instagram")
-            template(v-if="me.showChinaResourceInfo()")
```

Comment:
I proposed getting rid of social media column because we didn't really get any clicks in the US. Not sure if we still need it in China? cc @smallst

Reply:
We are ok to remove it for now, can bring it back in China if needed